websites/R/Raven Code/presence.ts

```diff
@@ -14,74 +14,81 @@ presence.on("UpdateData", async () => {
 	const presenceData: PresenceData = {
 			largeImageKey: "logo",
 			details: "Geziniyor",
+			buttons: [{ label: "Sayfaya git", url: document.location.href }],
 			startTimestamp,
 		},
-		{ pathname, search } = document.location,
+		{ pathname, search, host } = document.location,
 		searchParams = new URLSearchParams(search);
 
-	switch (pathname) {
-		case "/":
-			presenceData.details = "Anasayfayı görüntülüyor";
-			if (searchParams.has("user_id")) {
-				const image = document.querySelector<HTMLImageElement>(
-						"[data-premid-useravatar]"
-					)?.src,
-					userTag = document.querySelector<HTMLElement>(
-						"[data-premid-usertag]"
-					)?.innerText;
-				if (image && userTag) {
-					presenceData.details = "Kullanıcıyı görüntülüyor:";
-					presenceData.state = userTag;
-					presenceData.largeImageKey = image;
+	if (host === "docs.ravencode.live") {
+		presenceData.details = "docs.ravencode.live";
+		presenceData.state = document.title.replaceAll("-", "—");
+	} else if (host === "ravencode.live") {
+		switch (pathname) {
+			case "/":
+				presenceData.details = "Anasayfayı görüntülüyor";
+				if (searchParams.has("user_id")) {
+					const image = document.querySelector<HTMLImageElement>(
+							"[data-premid-useravatar]"
+						)?.src,
+						userTag = document.querySelector<HTMLElement>(
+							"[data-premid-usertag]"
+						)?.innerText;
+					if (image && userTag) {
+						presenceData.details = "Kullanıcıyı görüntülüyor:";
+						presenceData.state = userTag;
```

Comment:
This could be undefined due to the  check at the top.
```suggestion
						presenceData.state = userTag ?? "logo";
```

Reply:
This is never undefined.